src/main/java/org/gridsuite/voltageinit/server/service/VoltageInitNotificationService.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.voltageinit.server.service;
+
+import org.gridsuite.voltageinit.server.computation.service.NotificationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com
+ */
+@Service
+public class VoltageInitNotificationService extends NotificationService {
+
+    public static final String HEADER_REACTIVE_SLACKS_OVER_THRESHOLD = "REACTIVE_SLACKS_OVER_THRESHOLD";
+    public static final String HEADER_REACTIVE_SLACKS_THRESHOLD_VALUE = "reactiveSlacksThreshold";
+
+    @Autowired
+    public VoltageInitNotificationService(StreamBridge publisher) {
+        super(publisher);
+    }
+
```

Comment:
I get it that those string headers have to be put somewhere but I don't think that this is useful to create a child class just for this.

A class to put them in may be created but I don't think extending NotificationService brings anything here.

Reply:
I would either move them where they're used, or in a `Constants` class with others constants.